docker-compose.yml

```
@@ -0,0 +1,14 @@
+version: "3"
+
+services:
+  mysql:
+    image: mysql:8.0.32
+    environment:
+      - MYSQL_ROOT_PASSWORD
```

Default secrets can be visible if this `docker-compose.yml` is for development environment setup.
```suggestion
      MYSQL_ROOT_PASSWORD: ${DB_PASSWORD-root}
```
With [interpolation](https://docs.docker.com/compose/compose-file/12-interpolation/) it is possible to set defaults